packages/core/src/SlideBox/Arrow.styles.tsx

```diff
@@ -0,0 +1,23 @@
+import styled from 'styled-components'
```

Comment:
n-b Arrow.styled.tsx would be the convention.

Reply:
Updated  file name